server/controllers/student/addStudent.js

```diff
@@ -0,0 +1,24 @@
+const { insertStudent } = require('../../database/queries/addStudent');
+
+const addStudent = (req, res) => {
+  insertStudent({ ...req.body })
+    .then((student) => {
+      if (!student.rowCount) {
+        res.status(400).send({
+          error: 'Student dose not added',
+          data: null,
+        });
+      } else {
+        res.send({
+          error: null,
+          data: student.rows,
+        });
+      }
+    })
+    .catch(() => res.status(500).send({
```

Comment:
again you should make more specific error msg
```suggestion
    .catch(() => res.status(500).send({
```